tests/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -213,5 +224,6 @@ local utils = import 'templates/utils.libsonnet';
     transformer + v4_8 + convergence + timeouts.Hours(25) + tpuVm,
```

Comment:
Also add a convergence PJRT config. You don't have to test this one manually because it will take forever.